src/Common/src/System/Collections/Generic/EnumerableHelpers.cs

```diff
@@ -15,26 +17,22 @@ internal static T[] ToArray<T>(IEnumerable<T> source)
             return results;
         }
 
-        /// <summary>Converts an enumerable to an array using the same logic as does List{T}.</summary>
+        /// <summary>Converts an enumerable to an array using similar as does List{T}.</summary>
```

Comment:
Yes. Since it's not precisely the same, I went to replace "the same" with "similar", but I obviously must have selected more than I thought. Will fix.